tests/mutate/test_update_upsert.py

```diff
@@ -13,8 +13,9 @@ class TestUpdateVertex(NebulaTestSuite):
     @classmethod
     def prepare(self):
         resp = self.execute('CREATE SPACE myspace_test2(partition_num=1, replica_factor=1, vid_size=20);'
-                            'USE myspace_test2;'
-                            'CREATE TAG course(name string, credits int);'
+                            'USE myspace_test2;')
```

Comment:
I think this behavior is allowed. @CPWstatic wdyt?

Reply:
Even if want to support this feature, I don't plan to do it in this PR too. It will change too much logic about permission checking.

Reply:
I'm ok with it if we have plan to do it. I will add a new issue for it.